packages/design-system/src/Table/reset.css

```diff
@@ -0,0 +1,55 @@
+.rc-table.ads-v2-table-wrapper {
```

Comment:
Why not keep all styles in one place? You already have classes for all these overrides. I think it's better to move.

Reply:
So the reason for doing this here is, RC adds some divs on the fly and the classes we add is not applied on these. That's why I had to keep a seperate CSS file for this.